docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+
+Configuration Tabs
+===================
+The Control Center provides 3 tabs of configuration options: General, Tuning, and Advanced. These are found on the left side of the GUI, 
+and the selected tab is indicated in blue.
+
+.. image:: ../_static/control_center_pics/config_tabs.png
+
+.. note::
+    The configuration parameters presented for each module family and firmware style vary. Not all parameters shown in this example will be available for all modules and firmware styles.
+
+General
+-----------
+The general tab provides access to some more basic module configuration parameters. Parameters here help configure attributes necessary for properly controlling 
+your module with your desired behaviors.
+
+For example, the general tab is where you will find the :ref:`Motor Direction parameter <throttle_direction>`.
+
+Tuning
+-----------
+The tuning tab holds parameters geared towards affecting your module's performance. For example, this is where you find parameters that affect your module's PID gains.
+
+Advanced
+----------
+.. warning::
+    Changing parameters in the Advanced tab can compromise the safety features present on your module. Please use caution when changing parameters available in the Advanced tab.
+
+The advanced tab holds parameters that are meant for more experienced users. The advanced tab is where you find parameters such as your Module ID and UART Baud Rate.
+
+Configuration with Defaults
+============================
+IQ Control Center provides an easy method for setting several parameters at once with a system called Defaults. Defaults can be found in the general tab under MODULE PRE-SETTINGS
+
+.. image:: ../_static/control_center_pics/defaults.png
+
+There are two forms of defaults, one provided by default with the IQ Control Center software and one that is user generated.
+
+Using Vertiq Provided Default Settings
+-----------------------------------------
+The default settings provided with the Control Center are meant to help set parameters specific to certain battery and propeller settings configurations. 
+For example, you will find a default file called Vertiq2306_2200Kv_speed_APC6x4.json. This file provides tuning parameters for use of a 
+Vertiq 23-06 2200Kv module with speed firmware with an APC 6x4 propeller. In order to set these parameters, simply select the file that you would 
+like to use, then click Set.
+
+Using User Generated Default Settings
+-----------------------------------------
+User generated default settings allow you to easily share identical module configurations between multiple modules. For example, if you have parameters 
+that work for your module configuration (gains, modes, baud rate, etc.) that should be set on all of your modules, you would want to use a custom defaults file.
+
+.. note::
+    User generated Defaults files include the Module ID parameter. This means that setting up modules with custom Default files will result in collisions on 
```

Comment:
Not sure if the d in Defaults should be capitalized here, we don't capitalize it in other places we say defaults file